packages/app/package.json

```diff
@@ -36,13 +36,15 @@
     "@roadiehq/backstage-plugin-github-pull-requests": "0.3.0",
     "@roadiehq/backstage-plugin-travis-ci": "^0.2.3",
     "history": "^5.0.0",
+    "moment": "^2.27.0",
```

Comment:
Maybe we could use the much smaller [dayjs](https://github.com/iamkun/dayjs) instead :)

Reply:
We have a task to switch this out in our next iteration 👍

Reply:
I switched to dayjs for this PR in `app`, will look at updating cost-insights with the same soon.